lib/util/cache/repository/types.ts

```diff
@@ -25,6 +25,11 @@ export interface BranchUpgradeCache {
   sourceUrl?: string;
 }
 
+export interface PrCache {
+  fingerprint: string;
+  lastEdited: Date;
```

Comment:
Will this properly serialize to JSON later, or should we use a string here instead?

Reply:
When stringyfying the date will be converted to ISO string format and stored in cache. After parsing we receive it as a string so it creates type inconsistency, better to use a string I think.

Reply:
OK please go with string